flatflow/torch/nn/modules/attention.cu

```diff
@@ -0,0 +1,346 @@
+#include <cublas_v2.h>
+#include <cuda_runtime.h>
+#include <torch/extension.h>
+#include <torch/torch.h>
+#include <torch/types.h>
+#include <thread>
+#include <vector>
+
+#include "attention.h"
```

Comment:
Header files should be listed as descendants of the project's source directory without use of UNIX directory aliases.
That is, this should be ``#include "flatflow/aten/native/attention.h"``.